src/lime/_internal/backend/native/NativeApplication.hx

```diff
@@ -336,17 +337,18 @@ class NativeApplication
 			{
 				case MOUSE_DOWN:
 					window.onMouseDown.dispatch(mouseEventInfo.x, mouseEventInfo.y, mouseEventInfo.button);
+					window.clickCount = mouseEventInfo.clickCount;
 
 				case MOUSE_UP:
 					window.onMouseUp.dispatch(mouseEventInfo.x, mouseEventInfo.y, mouseEventInfo.button);
+					window.clickCount = mouseEventInfo.clickCount;
```

Comment:
Shouldn't we set `window.clickCount` _before_ dispatching the event? OpenFL records the value before returning from `__onLimeMouseDown` or `__onLimeMouseUp`.

Also probably worth setting it back to 0 after the event is done, to minimize cross-contamination. (Though even that won't stop it from showing up in the pending `MOUSE_MOVE` event.)

Reply:
We should, that's an oversight on my part. I'll fix that in the next commit. I didn't set it back to 0 for lime users (i wanted the value to still be accessible via window.clickCount when adding a listener to lime's mouse events)

Reply:
> i wanted the value to still be accessible via window.clickCount when adding a listener to lime's mouse events

But `onMouseDown` and `onMouseUp` are Lime's event dispatchers. Once those return, the mouse event listeners are done.